test/e2e-helm/e2e_helm_olm_test.go

```diff
@@ -53,7 +53,7 @@ var _ = Describe("Integrating Helm Projects with OLM", func() {
 			Expect(err).NotTo(HaveOccurred())
 
 			By("destroying the deployed package manifests-formatted operator")
-			cleanupPkgManCmd := exec.Command(tc.BinaryName, "cleanup", tc.ProjectName,
+			cleanupPkgManCmd := exec.Command(tc.BinaryName, "cleanup",
```

Comment:
Here you can see that it works. I will update all tests if we reach the consensus that it should be merged.